src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/fieldTransformers/BytesFieldTransformer.js

```diff
@@ -5,7 +5,7 @@ import type {FieldTransformer} from '../types';
 export default class BytesFieldTransformer implements FieldTransformer {
     transform(value: *): Node {
         if (value === undefined) {
-            return undefined;
+            return null;
```

Comment:
@trickreich That change should be ok, right? If not we would have to update the return value of the `transform` function. But `null` seems to be the more appropriate value in React, since a React component also can't return `undefined`. Asking for all the other transformers as well.

Reply:
Should be ok